komodoenv/__main__.py

```diff
@@ -56,13 +56,17 @@ def resolve_release(root: Path, name: str, no_update: bool = False) -> Tuple[Pat
     if not (root / name / "enable").is_file():
         sys.exit(f"'{root / name}' is not a valid komodo release")
 
+    env = os.environ.copy()
+    if "BASH_ENV" in env:
+        del env["BASH_ENV"]
```

Comment:
`BASH_ENV= bash -c '...'` sets `BASH_ENV` to an empty string rather than unsetting it. This is a small but significant difference.

```suggestion
    env["BASH_ENV"] = ""
```